benchmarks/Benchmarks.hs

```diff
@@ -258,6 +258,3 @@ main = do
                     , bench "untransMod"      $ nf LFull.untranslate (ann Nothing full)
                     ]
                 ]
-
-instance NFData SourceSpan where
```

Comment:
What's the reason for removing this? I would've presumed that this is necessary to run the benchmarks.

Reply:
This one is duplicated